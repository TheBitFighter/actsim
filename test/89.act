deftype value(bool a, b) { }

defproc test() {
  value y, x;
  chan(value) A;
  chp {
    [#A]; x := A; log ("got: ", x.a, " ", x.b) || y.a+, y.b-; A!y
  }
}
