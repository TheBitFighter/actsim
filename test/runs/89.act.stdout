[                  30] <>  got: 1 0
